apis/generate.go

```diff
@@ -22,6 +22,13 @@ limitations under the License.
 // Remove existing CRDs
 //go:generate rm -rf ../package/crds
 
+// Remove generated files
+//go:generate bash -c "rm -rf $(find . -iname 'zz_*')"
+//go:generate bash -c "rm -rf $(find ../internal/controller -iname 'zz_*')"
```

Comment:
I tried a few tricks to get the folders removed as well but for some reason the commands that work on my Terminal do not work here, like:
```
bash-c "rm -rf $(find ../internal/controller | sed 's/\.\.\/internal\/controller$//g' | grep -v '\/providerconfig' | grep -v '\/doc\.go')"
```

Reply:
@turkenh Just pushed a new commit that will clean everything up properly before the generation.